stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/TextFieldUI.kt

```diff
@@ -472,3 +476,6 @@ private fun Modifier.conditionallyClickable(onClick: (() -> Unit)?): Modifier {
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 const val DROPDOWN_MENU_CLICKABLE_TEST_TAG = "dropdown_menu_clickable"
+
+// Default size of Material Theme icons
+private const val LOADING_INDICATOR_SIZE = 24
```

Comment:
I initially tried using intrinsic sizing to have the loading indicator to be have the maximum height of the text field box but `CircularProgressIndicator` didn't respond it regardless of using `progressSemantics`. Using the default icon size for Material Theme instead which the icons have set.